pyfixest/estimation/multcomp.py

```diff
@@ -142,34 +150,48 @@ def rwolf(
     for i in range(S):
         model = models[i]
 
-        wildboot_res_df, bootstrapped_t_stats = model.wildboottest(
-            param=param,
-            reps=reps,
-            return_bootstrapped_t_stats=True,
-            seed=seed,  # all S iterations require the same bootstrap samples, hence seed needs to be reset
-        )
-        t_stats[i] = wildboot_res_df["t value"]
-        boot_t_stats[:, i] = bootstrapped_t_stats
+        if sampling_method == "wild-bootstrap":
+            wildboot_res_df, bootstrapped_t_stats = model.wildboottest(
+                param=param,
+                reps=reps,
+                return_bootstrapped_t_stats=True,
+                seed=seed,  # all S iterations require the same bootstrap samples, hence seed needs to be reset
+            )
+
+            t_stats[i] = wildboot_res_df["t value"]
+            boot_t_stats[:, i] = bootstrapped_t_stats
+
+        elif sampling_method == "ri":
+            model.ritest(
+                resampvar=param,
```

Comment:
Can you please provide a `rng` here? Each iteration of the procedure must run on the identical permutation of the data. I.e. within each loop, we need to apply the same rng. 

Unfortunately there is a small inconsistency in the `wildboottest` and `ritest` APIs - `ritest` requires a [np.RandomGenerator](https://numpy.org/doc/stable/reference/random/generator.html) while `wildboottest` is happy with a seed. 

E.g. you would have to add smth like

```python
rng = np.random.default_rng(seed) 
model.ritest(
          resampvar=param,
          ...
)
```